documentation/docs/30-advanced/10-advanced-routing.md

```diff
@@ -2,6 +2,38 @@
 title: Advanced routing
 ---
 
+## +loading.svelte
```

Comment:
Wasn't really sure if this belongs to "core concepts" or "advanced" - I put it in the latter